### PR TITLE
Added CNFWEntry sample configs for VPN

### PR DESCRIPTION
**Pull request by vivekvai**
Added CNFWEntry sample configs for VPN

### CODE REVIEW

File: samples/ra-vpn-sfc/redirector-lb/asaconfig-ipsec-enforcer.yaml [hunk 1]
```diff
@@ -0,0 +1,120 @@
+apiVersion: cnfw.cisco.com/v1
```
Comment (jaredhancock31) [inline samples/ra-vpn-sfc/redirector-lb/asaconfig-ipsec-enforcer.yaml]: regarding this directory structure...

would it make more sense to have it split on service-chained vs non service chain? example

* `samples/service-chaining/`
* `samples/service-chaining/vpn`
* `samples/service-chaining/cntd`
* `samples/default/`
* `samples/default/vpn`  <-- all the existing samples could move under this one

File: samples/ra-vpn-sfc/redirector-lb/ipv4subnetpool.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  name: ravpnpool
+  namespace: sfcn-system 
+  annotations:
+    # enables AWS Route Table integration for this pool
+    aws.cnfw.cisco.com/type: "route-table"
+    # EP's node interface index that will be used as route target
+    aws.cnfw.cisco.com/interface-index: "3"
+    # AWS Route Table ID that will be synced with assigned subnets.
+    # This should be the table that includes your inside networks, usually named "Inside Subnets"
+    aws.cnfw.cisco.com/route-table-id: <your_route_table_id>
```
Comment (jaredhancock31) [inline samples/ra-vpn-sfc/redirector-lb/ipv4subnetpool.yaml]: this was enhanced in 2.0 to allow users to just add `inside` as the value. that way they don't have to lookup the ID themselves

**Review by jaredhancock31 — CHANGES_REQUESTED** [review]

File: samples/ra-vpn-sfc/redirector-lb/ipv4subnetpool.yaml (hunk 2 above)
Comment (vivekvai) [reply samples/ra-vpn-sfc/redirector-lb/ipv4subnetpool.yaml]: Done

File: samples/ra-vpn-sfc/redirector-lb/asaconfig-ipsec-enforcer.yaml (hunk 1 above)
Comment (vivekvai) [reply samples/ra-vpn-sfc/redirector-lb/asaconfig-ipsec-enforcer.yaml]: After discussion with Santosh Kodali yesterday, we thought current structure is better one.

Comment (jaredhancock31) [reply samples/ra-vpn-sfc/redirector-lb/asaconfig-ipsec-enforcer.yaml]: what is the reasoning? if i'm someone looking to set up end to end service chaining, I'm bouncing around a lot of directories, especially considering where the CNTD stuff was merged to a couple days ago

Comment (santosh-kodali) [reply samples/ra-vpn-sfc/redirector-lb/asaconfig-ipsec-enforcer.yaml]: the proposed project structure makes sense to me

Comment (vivekvai) [reply samples/ra-vpn-sfc/redirector-lb/asaconfig-ipsec-enforcer.yaml]: Done

File: samples/service-chaining/vpn/ipv4subnetpool.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+apiVersion: cnfw.cisco.com/v1
+kind: IPv4SubnetPool
+metadata:
+  name: ravpnpool
+  namespace: sfcn-system 
+  annotations:
+    # enables AWS Route Table integration for this pool
+    aws.cnfw.cisco.com/type: "route-table"
+    # EP's node interface index that will be used as route target
+    aws.cnfw.cisco.com/interface-index: "3"
```
Comment (santosh-kodali) [inline samples/service-chaining/vpn/ipv4subnetpool.yaml]: index is wrong here, should be 1 
also, I believe if we don't provide index, it should pick it up automatically. 
please test and remove

File: samples/service-chaining/vpn/mock-user.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: userinfo 
+  namespace: sfcn-system
+stringData:
+  # RA VPN client details
+  username: jsmith 
+  password: cisco123
```
Comment (santosh-kodali) [inline samples/service-chaining/vpn/mock-user.yaml]: I believe this password will not work for 2.0 
security updates have taken place and need more secure pws 

has this config been tested?

File: samples/service-chaining/vpn/route53-ingress.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  name: vpn-redirector
+  namespace: sfcn-system
+spec:
+  # Change to your Route53 hosted zone
+  hostedZone: Z056XXXXXXXXXXXXXE0EK
+  # Change to your VPN domain name
+  recordSetName: vpn.domain.com
+  recordUpdate: DOMAIN
+  endpointSelector:
+      serviceRole: vpnredirector
```
Comment (santosh-kodali) [inline samples/service-chaining/vpn/route53-ingress.yaml]: providing index here will be more explicit and clear

File: samples/service-chaining/vpn/ipv4subnetpool.yaml (hunk 3 above)
Comment (vivekvai) [reply samples/service-chaining/vpn/ipv4subnetpool.yaml]: Done

File: samples/service-chaining/vpn/mock-user.yaml (hunk 4 above)
Comment (vivekvai) [reply samples/service-chaining/vpn/mock-user.yaml]: Done

File: samples/service-chaining/vpn/route53-ingress.yaml (hunk 5 above)
Comment (vivekvai) [reply samples/service-chaining/vpn/route53-ingress.yaml]: Done

**Review by santosh-kodali — APPROVED** [review]
lgtm

**Pull request merged**